jena-arq/src/main/java/org/apache/jena/riot/system/IRIResolver.java

```diff
@@ -129,391 +129,381 @@ private static void printErrorWarning(PrintStream ps, IRIFactory factory, int co
+//        // Not static - contains relative IRIs
+//        // Could split into absolute (static, global cached) and relative.
+//        private Cache<String, IRI> resolvedIRIs = CacheFactory.createCache(CacheSize);
+//
+//        /**
+//         * Construct an IRIResolver with base as the current working directory.
+//         */
+//        public IRIResolverNormal() {
+//            this((String)null);
+//        }
+//
+//        /**
+//         * Construct an IRIResolver with base determined by the argument URI. If
+//         * this is relative, it is relative against the current working
+//         * directory.
+//         *
+//         * @param baseStr
+//         * @throws RiotException
+//         *             If resulting base unparsable.
+//         */
+//        public IRIResolverNormal(String baseStr) {
+//            if ( baseStr == null )
+//                base = chooseBaseURI();
+//            else
+//                base = globalResolver.resolveSilent(baseStr);
+//        }
+//
+//        public IRIResolverNormal(IRI baseIRI) {
+//            if ( baseIRI == null )
+//                baseIRI = chooseBaseURI();
+//            base = baseIRI;
+//        }
+//
+//        @Override
+//        protected IRI getBaseIRI() {
+//            return base;
+//        }
+//
+//        @Override
+//        public IRI resolveSilent(String uriStr) {
+//            if ( resolvedIRIs == null )
+//                return resolveSilentNoCache(uriStr);
+//            else
+//                return resolveSilentCache(uriStr);
+//        }
+//
+//        private IRI resolveSilentNoCache(String uriStr) {
+//            IRI x = IRIResolver.iriFactory().create(uriStr);
+//            if ( SysRIOT.AbsURINoNormalization ) {
+//                // Always process "file:", even in strict mode.
+//                // file: is widely used in irregular forms.
+//                if ( x.isAbsolute() && ! uriStr.startsWith("file:") )
+//                    return x;
+//            }
+//            return base.create(x);
+//        }
+//
+//        private IRI resolveSilentCache(final String uriStr) {
+//            Callable<IRI> filler = () -> resolveSilentNoCache(uriStr);
+//            return resolvedIRIs.getOrFill(uriStr, filler);
+//        }
+//    }
+//
+//    /** Thread safe wrapper for an IRIResolver */
+//    static class IRIResolverSync extends IRIResolver
+//    {
+//        private final IRIResolver other;
+//
+//        IRIResolverSync(IRIResolver other) { this.other = other; }
+//        @Override
+//        synchronized
+//        protected IRI getBaseIRI() {
+//            return other.getBaseIRI();
+//        }
+//
+//        @Override
+//        synchronized
+//        public IRI resolve(String uriStr) {
+//            return other.resolve(uriStr);
+//        }
+//
+//        @Override
+//        synchronized
+//        public IRI resolveSilent(String uriStr) {
+//            return other.resolveSilent(uriStr);
+//        }
+//    }
```

Comment:
Has it been moved to the irix package/module? Could it be deleted?

Reply:
This is the old code largely not used but there are still a few places using jena-iri IRI directly based on the IRIFactory defined in this class. These need tracking down and individually checking - jena-iri is better at warnings about IRIs than the future replacement.

The commented methods are a temporary way someone is (unexpected) using IRIREsolver and can restore their code easily. Hopefully, that won't happen! 

There is a balance - to give the new code as much time in the default branch as possible (or pushing out the release date), the task of chasing down every detail isn't completed but is a manageable length now.